FAQ.md

```diff
@@ -0,0 +1,22 @@
+# Foire Aux Questions
+
+Cette page répertorie les questions fréquemment rencontrées au sujet de l'environnement technique d'OpenFisca.  
+N'hésitez pas à l'[enrichir](https://github.com/openfisca/tutorial/edit/master/FAQ.md) de vos expériences ! :slightly_smiling_face: 
+
+
+> Sous Windows, il est supposé que vous disposez d'un terminal bash (i.e [GitBash](https://gitforwindows.org) ou assimilé).
+
+## GIT
+
+| ERREUR | CONTEXTE | SOLUTION      |
+| ------ | -------- | ------------- |
+| `could not read from remote repository github` | Vous tentez de pousser votre travail en local sur le serveur avec : </br>`git push` | * S’assurer de l’existence d’une clef SSH avec `ls ~/.ssh/` </br>Attendu : une clef publique que nous nommerons `fichier_ssh.pub` et sa clef privée associée `fichier_ssh`. </br>* Le contenu de fichier_ssh.pub doit être référencé dans les settings du compte GitHub de l’usager. </br>* Ajouter la clef privée SSH à votre ssh-agent avec `ssh-add ~/.ssh/fichier_ssh` </br>Pour en savoir plus, voir l'[aide GitHub/SSH](https://help.github.com/articles/connecting-to-github-with-ssh/). |
```

Comment:
For beginners, wouldn't it be easier to guide them through https github connection ?

Reply:
Updated to explain that it's not a guide, only a solution for a user trying to `git push` with SSH. I don't remember the steps to do it in HTTPS but if you do @Anna-Livia, I will add them.